otherlibs/stdune-unstable/fpath.ml

```diff
@@ -1,11 +1,21 @@
 let is_root t = Filename.dirname t = t
 
-type mkdir_p =
+type mkdir_result =
   | Already_exists
   | Created
 
 let initial_cwd = Stdlib.Sys.getcwd ()
 
+let mkdir ?(perms = 0o777) t_s =
+  if is_root t_s then
```

Comment:
FWIW, `Unix.mkdir "/"` seems to work fine and raises `EEXIST`.
I guess the extra check won't hurt, especially if termination depends on this behavior, but doing it two times (here and in `mkdir_p`) when 0 could suffice seems unnecessary?

Reply:
Yeah, I'm not sure why we need this check. I just copied it from `mkdir_p` but I agree that it seems redundant.

Let me drop it.

Reply:
Done.